src/styles/main.scss

```diff
@@ -1,3 +1,60 @@
+$square-count: 31;
+$square-size: 100px;
+// $days: mon, tue, wed, thu, fri, sat, sun;
```

Comment:
move the variable to the new file:

`$cell-size: 100px;`
`$month-length: 31;`
`$days-in-a-week: 7;`
`$gap: 1px;`
`$padding: 10px;`
`$max-width: $days-in-a-week * $cell-size + 2 `
 ` *$padding + $gap * ($days-in-a-week - 1);`
`$week:`
  `"mon" 0,`
 ` "tue" 1,`
  `"wed" 2,`
  `"thu" 3,`
  `"fri" 4,`
 ` "sat" 5,`
  `"sun" 6;`